lib/config/optionsconfiguration.go

```diff
@@ -41,6 +41,7 @@ type OptionsConfiguration struct {
 	OverwriteRemoteDevNames bool     `xml:"overwriteRemoteDeviceNamesOnConnect" json:"overwriteRemoteDeviceNamesOnConnect" default:"false"`
 	TempIndexMinBlocks      int      `xml:"tempIndexMinBlocks" json:"tempIndexMinBlocks" default:"10"`
 	UnackedNotificationIDs  []string `xml:"unackedNotificationID" json:"unackedNotificationIDs"`
+	UnpauseOnStart          bool     `xml:"unpauseOnStart" json:"unpauseOnStart"`
```

Comment:
If this is set and you don't start with -(un)paused, it will unpause everything, just as it does in the current release. This just gives feature parity